content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
+```
+2) Take Jenkins instance Backup as a recovery option ::
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+
+cd /var/lib
+tar -cvzf jenkins_date.tar.gz jenkins/
+mv jenkins_date.tar.gz $HOME
+```
+
+3) Take The Backup Of the Current Jenkins Version Binary ::
+[source,bash]
+```
+#Take the backup of Jenkins current version binary using following commands:
+
+cd /usr/share/jenkins
+mv jenkins.war jenkins.war.old
+```
+
+4) Download The New LTS Jenkins Version ::
+[source,bash]
+```
+#Download the new version of Jenkins WAR file using the following commands:
+
+wget https://updates.jenkins-ci.org/latest/jenkins.war
+```
+
+5) Start The Jenkins Service ::
+[source,bash]
+```
+#Start the Jenkins service using the following command:
+service jenkins start
+```
+
+== Updating Based on Operating Systems
+---
+
+Red Hat / CentOS ::
+Beginning in CloudBees Jenkins Platform version 2.319.1.5, the epel-release package is required to upgrade CloudBees Jenkins Platform using an RPM.
```

Comment:
More of an info point, but CloudBees Jenkins Platform will be reaching [end of life](https://docs.cloudbees.com/docs/cloudbees-common/latest/maintenance-lifecycle#fixed-release) this coming summer (June 30, 2023) This may still be relevant but will not be supported in the future, fine to leave for now but just wanted to share that info!

Reply:
I think it's fine to leave in for now, as it still does apply. Thanks again @Vandit1604

Reply:
@dduportal It does seems weird i'll change it to [ **Jenkins** ] just now

Reply:
I have removed the commercial name does everything else looks good to you now ?

Reply:
Thanks a lot for this work!

Reply:
I am not really sure, as I do not use CentOS at all, sorry. I would defer to @MarkEWaite or @krisstern if they know the reason for this addon (is there an intent or is is a copy and paste from cloudbees doc ?)

at first sight, the version number (with 4 digits) should be changed (as the 4 digits versioning pattern is cloudbees only while jenkins LTS only has 3 digits)